client/app/visualizations/map/map-editor.html

```diff
@@ -70,6 +72,91 @@
       <select ng-options="tile.url as tile.name for tile in $ctrl.mapTiles"
         ng-model="$ctrl.options.mapTileUrl" class="form-control"></select>
     </div>
-  </div>
 
+    <h4>Marker</h4>
+
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" ng-model="$ctrl.options.customizeMarkers">
+        <i class="input-helper"></i>Override default style
+      </label>
+    </div>
+
+    <div class="row">
+      <div class="col-xs-6">
+        <div class="form-group">
+          <label>Shape</label>
+          <select ng-options="key as value for (key, value) in $ctrl.iconShapes"
+            ng-model="$ctrl.options.iconShape" class="form-control"
+            ng-disabled="!$ctrl.options.customizeMarkers"></select>
+        </div>
+      </div>
+      <div class="col-xs-6">
+        <div class="form-group">
+          <label>Icon Font
+            <span class="m-1-5" uib-popover-html="'Available <a href=&quot;https://fontawesome.com/v4.7.0/icons/&quot; target=&quot;_blank&quot;>icons.</a>'"
```

Comment:
I'd like to suggest a bit more verbosity. Sth like:
<img width="249" alt="Screen Shot 2019-06-15 at 17 31 06" src="https://user-images.githubusercontent.com/486954/59552754-60e38a00-8f93-11e9-9028-2597aad98c51.png">

Reply:
Much better 😄